### PR TITLE
Sorting method sorting child pages

### PR DESCRIPTION
**Pull request by Eskilmar**

### CODE REVIEW

File: src/components/generic/Header.tsx [hunk 1]
```diff
@@ -336,6 +336,26 @@ class Header extends React.Component<Props, State> {
     }
   }
 
+  /**
+   * Method for comparing menuorder from CustomPage. Used to sort pages
+   * 
+   */
```
Comment (terolamminpaa) [inline src/components/generic/Header.tsx:342]: Add parameters and return value to doc and also specify method return value type. For example `private func = (): string => { return "hello" }` has return type of string.

**Review by terolamminpaa — CHANGES_REQUESTED** [review]

File: src/components/generic/Header.tsx (hunk 1 above)
Comment (Eskilmar) [reply src/components/generic/Header.tsx]: Made changes

File: src/components/generic/Header.tsx [hunk 2]
```diff
@@ -336,6 +336,30 @@ class Header extends React.Component<Props, State> {
     }
   }
 
+  /**
+   * Method for comparing menuorder numbers from CustomPage.
+   * 
+   * @param CustomPage object
+   * @param CustomPage object
```
Comment (terolamminpaa) [inline src/components/generic/Header.tsx:343]:
```suggestion
   * @param a CustomPage object
   * @param b CustomPage object
```

**Review by terolamminpaa — APPROVED** [review]
You can merge this yourself after you fix that little detail

**Pull request merged**